app.js

```diff
@@ -288,6 +291,21 @@ $(document).ready(function() { //  Beginning of jQuery
         .text(snapshot.val().name)
 
       $("#topList").append($("<li>").text(snapshot.val().name + "(" + snapshot.val().visits * (-1) + " searches)"));
+
+      // Jason Added
+      
+      console.log(snapshot.val());
+      var topArtist = {
```

Comment:
this is totally unnecessary, look at my pull request

Reply:
since you deleted my pr i guess you can't look, here's the problem, creating topArtist is totally unnecessary, simply use the artist i created above